### PR TITLE
KEYCLOAK-11757 Make sure "other" tests pass

### PR DESCRIPTION
**Pull request by tkyjovsk**
- fixed FlowsTest, PasswordPolicyTest
- fixed AbstractConsoleTest which was broken by fixing KEYCLOAK-11853

### CODE REVIEW

**Review by mposolda — APPROVED** [review]
@tkyjovsk Thanks!

File: testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/AbstractConsoleTest.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 import org.junit.Before;
 import org.keycloak.representations.idm.UserRepresentation;
 import org.keycloak.testsuite.AbstractAuthTest;
+import static org.keycloak.testsuite.admin.Users.setPasswordFor;
```
Comment (fperot74) [inline testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/AbstractConsoleTest.java]: Please move this import static with other ones (line 36-38)

File: testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authentication/FlowsTest.java [hunk 2]
```diff
@@ -214,20 +205,24 @@ public void navigationTest() {
         assertAlertSuccess();
 
         List<String> expectedOrder = new ArrayList<>();
-        Collections.addAll(expectedOrder, "Kerberos", "Cookie", "Copy Of Browser Forms", "OTP Form",
-                                          "Username Password Form", "Identity Provider Redirector");
-
+        Collections.addAll(expectedOrder,
```
Comment (fperot74) [inline testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authentication/FlowsTest.java]: You can use a single instruction: `List<String> expectedOrder = Arrays.asList(value1, value2, ..., valueN);`

File: testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authentication/PasswordPolicyTest.java [hunk 3]
```diff
@@ -18,7 +18,9 @@
 package org.keycloak.testsuite.console.authentication;
 
 import org.jboss.arquillian.graphene.page.Page;
+import static org.junit.Assert.assertTrue;
```
Comment (fperot74) [inline testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authentication/PasswordPolicyTest.java]: Move `import static` after other `import`

**Review by fperot74 — COMMENTED** [review]
Functionally: approved
I suggested some "decoration" changes.

**Comment by tkyjovsk** [comment]
@fperot74 Thanks for the review, Francis. I think your comments should be addressed in the latest commit `b7c2ed2`.

**Review by fperot74 — APPROVED** [review]

**Review by AlistairDoswald — APPROVED** [review]

**Pull request merged**